src/driver/Firestore/InProcessFirestore.ts

```diff
@@ -215,7 +216,7 @@ interface IIdItem {
 
 interface IQueryBuilder {
     maps: Array<(item: IItem) => IItem>
-    filters: Array<(item: IItem) => boolean>
+    filters: Array<(idItem: IIdItem) => boolean>
```

Comment:
this is a breaking change right? Before it wasn't require to adhere to have an `id` and `item`. But now it does?

If so the version becomes 1.0.0 to follow SemVer.

Reply:
The public interface hasn't changed, this array is an internal component. Or at least, these internal fields should never have been exposed akin to the real `Firestore.Query` data type. As we we are still officially pre-release (0.x), I'm of the mind set this isn't a breaking change?

Reply:
![image](https://user-images.githubusercontent.com/5040901/104012521-d5020600-51a7-11eb-967a-9f52e990d5d2.png)

Reply:
If it's internal. Woop